lib/plugins/aws/lib/naming.js

```diff
@@ -22,21 +21,17 @@ module.exports = {
     return /^ServiceEndpoint/;
   },
 
-  /*
-   * Stack Naming
-   */
+  // Stack
   getStackName() {
     return `${this.sdk.serverless.service.service}-${this.sdk.getStage()}`;
   },
 
-  getLogicalLogGroupName(functionName) {
-    return `${this.getNormalizedLambdaName(functionName)}LogGroup`;
+  getLogGroupLogicalId(functionName) {
+    return `${this.getNormalizedFunctionName(functionName)}LogGroup`;
```

Comment:
Why does this not use `lambda` but `function` since were using `lambda` everywhere else?